src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static MIN_HEALTH = 1;
```

Comment:
The static variable `MIN_HEALTH` is set to 1, but according to the task description, an animal should be considered dead and removed from the alive array if its health becomes less than or equal to 0. The current value should be changed to 0 to reflect this condition.